File: RP3/Desafio4.java
import java.util.Scanner;

public class Desafio4 {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {

            int numeroSecreto = 7;
            int palpite = 0;

            while (palpite != numeroSecreto) {
                System.out.print("Adivinhe o número: ");
                palpite = sc.nextInt();

                if (palpite > numeroSecreto) {
                    System.out.println("Tente um número menor!");
                } else if (palpite < numeroSecreto) {
                    System.out.println("Tente um número maior!");
                }
            }

            System.out.println("Parabéns! Você acertou!");
        }
    }
}

File: RP3/Desafio4.py
numero_secreto = 7
palpite = 0

while palpite != numero_secreto:
    palpite = int(input("Adivinhe o número: "))

    if palpite > numero_secreto:
        print("Tente um número menor!")
    elif palpite < numero_secreto:
        print("Tente um número maior!")

print("Parabéns! Você acertou!")